.github/workflows/release.yml

```diff
@@ -88,3 +88,63 @@ jobs:
           package: ./cli/package.json
           check-version: true
           access: public
+  demo:
+    needs: release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup Node
+        uses: actions/setup-node@v2
+        with:
+          node-version: "16"
+      - name: Install dolos
+        run: |
+          npm install -g @dodona/dolos
+          dolos --version
+      - name: Setup SSH
```

Comment:
Any reason why you do this manually instead of using `shimataro/ssh-key-action@v2` ?

Reply:
Simplicity, mostly. I've looked at that action but it is essentially equal to the five lines in this step. Adding an extra dependency on an action for such a simple task  seemed like overkill to me.